apps/web/src/components/forms/team-document-settings.tsx

```
@@ -0,0 +1,155 @@
+      ...data,
+      teamId: team.id,
+    });
+  };
+
+  const mapVisibilityToRole = (visibility: string): string => {
+    switch (visibility) {
+      case DocumentVisibility.ADMIN:
+        return 'ADMIN';
+      case DocumentVisibility.MANAGER_AND_ABOVE:
+        return 'MANAGER';
+      case DocumentVisibility.EVERYONE:
```

Biome

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)